classes/abstracts/ActionScheduler_Store.php

```diff
@@ -449,10 +449,11 @@ public function get_status_labels() {
 	 */
 	public function has_pending_actions_due() {
 		$pending_actions = $this->query_actions( array(
-			'date'    => as_get_datetime_object(),
-			'status'  => self::STATUS_PENDING,
-			'orderby' => 'none',
-		) );
+			'per_page' => 1,
+			'date'     => as_get_datetime_object(),
+			'status'   => ActionScheduler_Store::STATUS_PENDING,
```

Comment:
```suggestion
			'status'   => self::STATUS_PENDING,
```

Looks like this should be updated after [this other change](https://github.com/woocommerce/action-scheduler/commit/b69152038912f0a62de173d3d13587969174fcbc) was committed.